### PR TITLE
[BL-411] Bump Go to 1.13

### PR DESCRIPTION
**Pull request by zimnx**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.11.4
+      - image: circleci/golang:1.13
```
Comment (tomasz-michalski) [inline .circleci/config.yml]: `golang:1.11.4` is still used in `test` job below

**Comment by zimnx** [comment]
> I think it would be good to run `go mod tidy`

done

File: .circleci/config.yml (hunk 1 above)
Comment (zimnx) [reply .circleci/config.yml]: done

File: go.mod [hunk 2]
```diff
@@ -71,16 +71,20 @@ require (
...
 	gopkg.in/mgo.v2 v2.0.0-20180705113604-9856a29383ce // indirect
 	gopkg.in/sourcemap.v1 v1.0.5 // indirect
+	gopkg.in/stretchr/testify.v1 v1.0.0-00010101000000-000000000000 // indirect
 	gopkg.in/yaml.v2 v2.2.2
 )
 
-replace github.com/onsi/ginkgo v1.6.0 => github.com/cloudwan/ginkgo v1.6.0-0.20190213151947-95174e8d10cd
+replace github.com/onsi/ginkgo => github.com/cloudwan/ginkgo v1.6.1-0.20190213151947-95174e8d10cd
+
+replace gopkg.in/stretchr/testify.v1 => github.com/stretchr/testify v1.4.0
```
Comment (tomasz-michalski) [inline go.mod]: Why is this needed? I removed all references to `testify` from `go.mod` and all tests passed.
